custom_commands.py

```diff
@@ -108,10 +108,10 @@ def __repr__(self):
     def bind(self):
         return None
 
-    def overwrite(self, overwrite):
+    def force(self, overwrite):
```

Comment:
Interesting - this code comes directly from Snowflake, I did not see that this parameter exists and set the "force" option manually.

Could you please also rename the parameter `overwrite` to `force`, and change the text in the TypeError?

Reply:
Done. The overwrite parameter cause the snowflake sql error. So just renamed that to force to match the snowflake documentation.

Reply:
TypeError comment is fixed.